### PR TITLE
Added code for AMA

### PR DESCRIPTION
**Pull request by omenocal**

### CODE REVIEW

File: samples/starterKit/config/local.json.example [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "server": {
+    "port": 3000,
+    "hostSkill": true
+  }
+  "opearlo": {
```
Comment (armonge) [inline samples/starterKit/config/local.json.example]: opearlo?

File: index.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+/**
+ * Voxa-Opearlo integration
```
Comment (wuelcas) [inline index.js]: This should be 'Voxa-AmazonMobileAnalytics', not 'Voxa-Opearlo', right?

File: samples/starterKit/skill/MainStateMachine.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+const views = require('./views');
+const variables = require('./variables');
+const states = require('./states');
+const voxaAma = require('voxa-amazon-mobile-analytics');
+const config = require('../config');
+
+const skill = new Voxa({ variables, views });
+
+states.register(skill);
+voxaAma(skill, config.ama);
```
Comment (armonge) [inline samples/starterKit/skill/MainStateMachine.js]: do we have examples of how this config looks like?

File: samples/starterKit/README.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
+# Voxa
```
Comment (armonge) [inline samples/starterKit/README.md:1]: In general i'd like to takes this as an oportunnity to make a simpler example, like, something really minimal to just show the minimum required voxa project with analytics

File: package.json [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+    "analytics",
+    "ama",
+    "amazon",
+    "mobile",
+    "analytics",
+    "amazon mobile analytics"
+  ],
+  "author": "Rain Agency <npm@rain.agency> (http://rain.agency)",
+  "engines": {
+    "node": ">=4.3.2 <=6.10"
```
Comment (armonge) [inline package.json]: let's stop restricting the maximum node version required, if they want to host voxa on an ec2 instance with node 8 it's their deal

File: samples/starterKit/skill/index.js [hunk 6]
```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+// Include the state machine module, the state machine,
+// the responses and variables to be used in this skill
+const skill = require('./MainStateMachine');
+require('./states');
+
+exports.handler = function handler(event, context, callback) {
```
Comment (armonge) [inline samples/starterKit/skill/index.js]: voxa now supports a simpler version of this

File: samples/starterKit/www/server.js [hunk 7]
```diff
@@ -0,0 +1,32 @@
+'use strict';
```
Comment (armonge) [inline samples/starterKit/www/server.js]: we can probably get rid of all the www stuff for this example

File: samples/starterKit/skill/MainStateMachine.js (hunk 3 above)
Comment (omenocal) [reply samples/starterKit/skill/MainStateMachine.js]: @armonge in the README:
```
const amaConfig = {
  appId: 'appId',
  appTitle: 'appTitle',
  make: 'Amazon',
  platform: 'Alexa',
  cognitoIdentityPoolId: 'cognitoIdentityPoolId',
  suppressSending: false, // A flag to supress sending hits. Useful while developing on the skill
};
```

File: samples/starterKit/.gitignore [hunk 8]
```diff
@@ -0,0 +1,85 @@
+aws-config.js
+#### OSX
+*.DS_Store
```
Comment (rmberrios) [inline samples/starterKit/.gitignore:3]: Could we remove the sample dir from this project?

**Review by rmberrios — CHANGES_REQUESTED** [review]

File: samples/starterKit/.gitignore (hunk 8 above)
Comment (armonge) [reply samples/starterKit/.gitignore:3]: i think an example dir would be great, something that shows how to use the plugin. But it should be a very minimal example. Even smaller than this. Maybe something like the package.json and an index.js file, with variables and views in the same file, comments?

**Review by rmberrios — APPROVED** [review]

**Review by armonge — APPROVED** [review]

File: samples/starterKit/.gitignore (hunk 8 above)
Comment (omenocal) [reply samples/starterKit/.gitignore:3]: Just made it that way, based on the voxa example in the README

**Pull request merged**